README.md

```diff
@@ -43,12 +43,10 @@ billing = "TODO"
 watchers = {
     "w1" = {
         "interval"      : "* */10 * * *",
-        "function"      : "TODO",
-        "topic"         : "TODO" ,
-        "scheduler"     : "TODO",
-        "function_url"  : "TODO",
+        "name"          : "TODO",
+        "target_url"    : "TODO",
         "email"         : "TODO",
-        "string_target" : "TODO",
+        "target_string" : "TODO",
```

Comment:
I simplified the setup by using the "name" variable to create all the resources. Ideally we can even use the "w1" value.

Reply:
I will be adding this after I merge this PR! This should be possible by using `each.key` instead of `each.value.name` when defining the `module watcher` in `terraform/main.tf` I gather?